api_fetch.js

```diff
@@ -1,131 +1,130 @@
-        descriptionElement.textContent = "Description: " + this.data.description;
-        container.append(descriptionElement);
-
-        // Create Book room button for each Challenge card
-        const button = document.createElement("button");
-        button.textContent = "Book this room";
-        button.dataset.challengeId = this.data.id;
-        container.append(button);
-
-        // Listen to button and forward challenge id to show title in booking modal
-        button.addEventListener("click", function (event) {
-            const challengeId = event.currentTarget.dataset.challengeId;
-            console.log("Challenge id:", challengeId);
-            document.getElementsByClassName("modal__stepOne")[0].style.display = "block";
-
-            const selectedChallenge = allChallenges.find(challenge => challenge.data.id === parseInt(challengeId));
-
-            const bookRoomTitle = document.createElement("h1");
-            bookRoomTitle.textContent = "Book Room: \"" + selectedChallenge.data.title + "\" (Step 1)";
-
-            const modalStepOne = document.querySelector("#challenge__title");
-            modalStepOne.append(bookRoomTitle);
-
-
-        });
-
-
-        return container;
-    }
+    // Create Book room button for each Challenge card
+    const button = document.createElement("button");
+    button.textContent = "Book this room";
+    button.dataset.challengeId = this.data.id;
+    container.append(button);
+
+    // Listen to button and forward challenge id to show title in booking modal
+    button.addEventListener("click", function (event) {
+      const challengeId = event.currentTarget.dataset.challengeId;
+      //document.getElementsByClassName("modal__stepOne")[0].style.display = "block";
+      displayModalStepOne(modal1);
+
+      selectedChallenge = allChallenges.find(
+        (challenge) => challenge.data.id === parseInt(challengeId)
+      );
+
+      const bookRoomTitle = document.querySelector("#modal1__title");
+      bookRoomTitle.textContent =
+        'Book Room: "' + selectedChallenge.data.title + '" (Step 1)';
+    });
+    return container;
+  }
 }
 
-
 // Request Challenge data from API and Create Challenge objects and put them in allChallenges array
 class APIadapter {
-    async getAllChallenges() {
-        const url = 'https://lernia-sjj-assignments.vercel.app/api/challenges';
-        const response = await fetch(url);
-        const payload = await response.json();
+  async getAllChallenges() {
+    const url = "https://lernia-sjj-assignments.vercel.app/api/challenges";
+    const response = await fetch(url);
+    const payload = await response.json();
 
-        allChallenges = payload.challenges.map((challengeData) => new Challenge(challengeData));
+    allChallenges = payload.challenges.map(
+      (challengeData) => new Challenge(challengeData)
+    );
 
-        return allChallenges;
-    }
+    return allChallenges;
+  }
```

Comment:
Here you're making changes only to whitespace, probably because your team has not agreed on a set of rules for whitespace (and ideally configured your editors to conform to those rules). This means that the git diff shows changes that are actually not relevant to the functionality of the code (just the style of the code).

More changes means more things for a reviewer to keep in their head, and a greater risk that they will miss something significant.

There's a feature in GitHub to hide whitespace-only changes in a PR diff when reviewing, but it's best to avoid the issue completely by agreeing on rules within the team.